app/src/main/java/com/sunasterisk/moviedb_51/ui/home/HomeFragment.kt

```diff
@@ -117,8 +119,26 @@ class HomeFragment : Fragment(), TabLayout.OnTabSelectedListener,
 
     private fun initView() {
         binding.run {
-            genresRecyclerView.adapter = genreAdapter
-            moviesByCategoryRecyclerView.adapter = movieCategoryAdapter
+            genresRecyclerView.adapter = genreAdapter.apply {
+                onItemClick = { genres ->
+                    activity?.addFragment(
+                        R.id.mainFrameLayout,
+                        MoviesFragment.getInstance(
+                            MoviesTypes.GENRES.value,
+                            genres.genresID.toString(),
+                            genres.genresName
+                        )
+                    )
```

Comment:
Tạo biến fragment cho đỡ cồng kềnh 
```suggestion
                    val fragment = ... 
                    activity?.addFragment(R.id.mainFrameLayout, fragment) 
```